src/resourceMeta.ts

```diff
@@ -17,8 +17,9 @@ export interface Meta {
     versionId: string;
     lastUpdated: string;
     source?: string;
-    profile?: Coding;
-    tag?: Coding;
+    profile?: any | any[];

```

Comment:
Only functional code change; everything else is re-arranging & dependency update